2020/summer_meeting/session_four_incident_response.md

```diff
@@ -0,0 +1,101 @@
+# Session 4: Incident Response
+
+Notetaker 1: Jeremy Prevost
+
+Notetaker 2: James Griffin
+
+## Initial Topics
+
+- Managing your team members in an incident response
+- Testing ability to restore services
+- communication outward
+- Response template / report outs
+- How do we learn from incidents
+
+## Managing your team members in an incident response
+- Overview of what people are doing
+    - Emergency services incident command system as background. First person in scene is incident commander until someone with more experience shows up. Are others in library tech doing similar / different things?
```

Comment:
instead of "more experience" I think this should be "more managerial responsibility"